ui/src/timeMachine/components/view_options/ScatterOptions.tsx

```diff
@@ -0,0 +1,184 @@
+// Libraries
+import React, {SFC} from 'react'
+import {connect} from 'react-redux'
+
+// Components
+import {Form, Input, Grid} from '@influxdata/clockface'
+import {Dropdown, MultiSelectDropdown} from 'src/clockface'
+import ColorSchemeDropdown from 'src/shared/components/ColorSchemeDropdown'
+import AxisAffixes from 'src/timeMachine/components/view_options/AxisAffixes'
+
+// Actions
+import {
+  setFillColumns,
+  setSymbolColumns,
+  setColors,
+  setYAxisLabel,
+  setXAxisLabel,
+  setAxisPrefix,
+  setAxisSuffix,
+} from 'src/timeMachine/actions'
+
+// Utils
+import {
+  getGroupableColumns,
+  getFillColumnsSelection,
+  getSymbolColumnsSelection,
+} from 'src/timeMachine/selectors'
+
+// Types
+import {ComponentStatus} from '@influxdata/clockface'
+import {Color} from 'src/types/colors'
+import {AppState} from 'src/types'
+
+interface StateProps {
+  fillColumns: string[]
+  symbolColumns: string[]
+  availableGroupColumns: string[]
+}
+
+interface DispatchProps {
+  onSetFillColumns: typeof setFillColumns
+  onSetSymbolColumns: typeof setSymbolColumns
+  onSetColors: typeof setColors
+  onSetYAxisLabel: typeof setYAxisLabel
+  onSetXAxisLabel: typeof setXAxisLabel
+  onUpdateAxisSuffix: typeof setAxisSuffix
+  onUpdateAxisPrefix: typeof setAxisPrefix
+}
+
+interface OwnProps {
+  xColumn: string
+  yColumn: string
+  fillColumns: string[]
+  symbolColumns: string[]
+  xDomain: [number, number]
+  yDomain: [number, number]
+  xAxisLabel: string
+  yAxisLabel: string
+  xPrefix: string
+  xSuffix: string
+  yPrefix: string
+  ySuffix: string
+  colors: Color[]
+  showNoteWhenEmpty: boolean
+}
+
+type Props = OwnProps & DispatchProps & StateProps
+
+const ScatterOptions: SFC<Props> = props => {
+  const {
+    fillColumns,
+    symbolColumns,
+    colors,
+    availableGroupColumns,
+    yAxisLabel,
+    xAxisLabel,
+    onSetFillColumns,
+    onSetSymbolColumns,
+    onSetColors,
+    onSetYAxisLabel,
+    onSetXAxisLabel,
+    yPrefix,
+    ySuffix,
+    onUpdateAxisSuffix,
+    onUpdateAxisPrefix,
+  } = props
+
+  const groupDropdownStatus = availableGroupColumns.length
+    ? ComponentStatus.Default
+    : ComponentStatus.Disabled
+
+  return (
+    <Grid.Column>
+      <h4 className="view-options--header">Customize Scatter Plot</h4>
+      <h5 className="view-options--header">Data</h5>
+
+      <Form.Element label="Symbol domain column">
```

Comment:
Maybe "Symbol Columns"? "Point Shape"? I think "Symbol domain column" is confusing, but I don't know what would be better.